golem_messages/datastructures/tasks.py

```diff
@@ -55,6 +55,7 @@ class TaskHeader(datastructures.Container):
                 fail_msg="Subtask timeout is less than 0",
             ),
         ),
+        'task_type': (validators.validate_varchar128, ),
```

Comment:
Shouldn't it be an enum?

Reply:
That would require adjusting this field every time we add a new app, which IMHO adds more work then profit